OPERATOR.rst

```diff
@@ -313,6 +344,27 @@ For GitLab or ClamAV updates, use the ``--no-restart`` flag in order to leave
 the instance stopped after the snapshot has been created. There is no point in
 starting the instance only to have the update terminate it again.
 
+Update GitLab via ``yum update``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To apply updates to the Linux instance running GitLab, SSH into the system and
```

Comment:
How do we do this for production instances where no operator has SSH access?

Reply:
I've added a prerequisite for this instructions, requiring SSH access to the instances where the updates are planned. Otherwise, the operator should request for the project lead to run the yum update step and confirmation of when is it clear to proceed with the reboot.